tests/BrandingClientTest.php

```diff
@@ -261,33 +250,6 @@ public function cachingTimesDataProvider()
         ];
     }
 
```

Comment:
Flush cache is no longer done within branding or orbit, now is using the flushing mechanims provided by the caching library